src/silx/gui/plot/actions/histogram.py

```diff
@@ -275,6 +275,10 @@ def __init__(self, *args, **kwargs):
             self.__updateHistogramFromControls)
         self.__rangeSlider.sigValueChanged.connect(self.__rangeChanged)
         controlsLayout.addWidget(self.__rangeSlider)
+        self.__weightCheckBox = qt.QCheckBox(self)
+        self.__weightCheckBox.setText("Use weights")
+        self.__weightCheckBox.stateChanged[int].connect(self.__weightChanged)
```

Comment:
small detail but I think the 'silx policy' would be more to use the 'clicked' signal no @t20100 ?

Reply:
For a checkbox that is going to be accessed through the user interface, the two signals are indeed the same.

However, stateChanged is emitted when the checkbox changes either programmatically or via the interface. clicked is not emitted when the checkbox is modified via the interface.

So, in this case, I do not care.

Reply:
https://doc.qt.io/qt-6/qabstractbutton.html#clicked